### DISCUSSION TITLE
Can't jump to list of forks from project home

### DISCUSSION THREAD

**Discussion by idbrii** [community/community#9660] [Other Feature Feedback, Questions, & Ideas]
The redesigned buttons in the top right of a project no longer allow quick jumping to the list of forks. You used to be able to click on the number to jump to the list of forks and click the word "fork" to fork the project.

![image](https://user-images.githubusercontent.com/43559/148187826-5dd977e3-2e71-4b50-98de-6ca5c2286599.png)

Now you have to dig into Insights to find the list of forks. (Which I'd never have found except I knew the list existed so I clicked everywhere.)

**Request:** Could you add the link to the list of forks to the dialog that pops up when you click "Fork"? Or provide another easy way to find the forks?

**Comment by tinyhaggis** [comment]
The forks are listed on the main page of the repo in the right side About section, but you need to be on the main page to see that panel.  I agree that this is a bad design decision since I also used to view forks quickly by clicking the number.
